appsmithorg/appsmith
# fix: Adjust status details style in API Response view

## Description
 
- status details in section in the API Response view needs Adjustment

Fixes #10959

## Type of change

- Bug fix (non-breaking change which fixes an issue)

## How Has This Been Tested?

- Create a GET Request
- Check the status details.

![Screenshot 2022-02-17 at 09 59 10](https://user-images.githubusercontent.com/25982036/154441155-2eb69161-11f8-483b-a730-21faf176b410.png)
![Screenshot 2022-02-17 at 09 59 17](https://user-images.githubusercontent.com/25982036/154441164-b2b396dc-72ee-4316-bf91-8c1ae759e7bd.png)



## Checklist:







## Test coverage results :test_tube:
<details><summary>:red_circle: Total coverage has decreased</summary>


    // Code coverage diff between base branch:release and head branch: fix/status-code-style-adjustment 
Status | File | % Stmts | % Branch | % Funcs | % Lines 
 -----|-----|---------|----------|---------|------ 
 :red_circle: | total | 56.5 **(-0.02)** | 37.97 **(-0.03)** | 35.84 **(-0.02)** | 56.74 **(-0.02)**
 :red_circle: | app/client/src/components/editorComponents/ApiResponseView.tsx | 45.22 **(-4.3)** | 28.32 **(-2.16)** | 0 **(0)** | 46.79 **(-4.21)**
 :red_circle: | app/client/src/selectors/commentsSelectors.ts | 83.61 **(-1.64)** | 61.76 **(-2.95)** | 73.33 **(0)** | 88.24 **(-2.35)**
 :red_circle: | app/client/src/utils/WorkerUtil.ts | 89.76 **(0)** | 70.59 **(-1.96)** | 100 **(0)** | 93.33 **(0)**
 :red_circle: | app/client/src/utils/autocomplete/TernServer.ts | 52.71 **(-0.23)** | 40.83 **(-0.84)** | 36.21 **(0)** | 56.74 **(-0.25)**</details>
